jwst/regtest/test_associations_sdp_pools.py

```diff
@@ -70,25 +159,25 @@ def _assoc_sdp_against_standard(rtdata, resource_tracker, request, pool_args):
         ("jw00809_20220124t091748_pool", []),
         ("jw00818_20230407t030411_pool", []),
         ("jw00839_20221220t025418_pool", ["-i", "o002", "c1000"]),
+        ("jw01122_uniq_lamp_optical_path_pool", []),  # NRS_FSS_VALID_LAMP_OPTICAL_PATHS
```

Comment:
Not sure where to document this but for now here. We made this file from this code below because the original pool has over 2k rows.

```python
from astropy.table import unique, Table

f1 = "jw01122_20250330t072318_pool.csv"
t1 = Table.read(f1, delimiter="|", format="ascii")
selected_cols = ['GRATING', 'LAMP', 'DETECTOR', 'OPMODE']
uniq_combos = unique(t1, keys=selected_cols)
uniq_combos.write("jw01122_uniq_lamp_optical_path_pool.csv", delimiter="|", format="ascii")
```